src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1270,10 +1387,13 @@ type Place {
     rentalVehicle: RentalVehicle
 
     """The bike parking related to the place"""
-    bikePark: BikePark
+    bikePark: BikePark @deprecated("bikePark is deprecated. Use vehicleParking instead.")
```

Comment:
The schema build fails currently. I think you have to add "reason:" prefix to the deprecation comments.
```suggestion
    bikePark: BikePark @deprecated(reason: "bikePark is deprecated. Use vehicleParking instead.")
```